MusicBot.yaml

```
@@ -117,6 +117,105 @@ paths:
           $ref: "#/components/responses/Unauthenticated"
         403:
           $ref: "#/components/responses/Unauthorized"
+  "/user/{username}":
+    get:
+      operationId: getUser
+      summary: Retrieves information about a user
+      description: >
+        Returns information about the user specified in the path.
+        Permission information about the user reflects the current state of the bot, permissions saved in still valid JWT tokens may vary.
+      parameters:
+        - name: username
+          in: path
+          schema:
+            type: string
+          description: The user to retrieve. Usernames are case-insensitive
+          required: true
+      responses:
+        200:
+          description: Information about the user
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserInfo"
+        401:
+          $ref: "#/components/responses/Unauthenticated"
+        403:
+          $ref: "#/components/responses/Unauthorized"
+        404:
+          description: The user does not exist
+  "/user/{username}/permission"
+    put:
+      operationId: setUserPermissions
+      summary: Set a user's permissions
+      description: >
+        Updates a user's permissions.
+        Permissions omitted in the request will be dropped if owned by the user previously.
+        The update will reach the user, when he updates his JWT token.
```

I think that's a very German comma

```suggestion
        The update will reach the user when he updates his JWT token.
```